packer/sagebio/sagebio-base-ubuntu-bionic-LATEST/playbook.yaml

```diff
@@ -18,10 +18,11 @@
         state: present
 
     # awscli is on aws linux2 distro by default
-    - name: install awscli
+    - name: install awscli and SSM
       snap:
         name:
           - aws-cli
+          - amazon-ssm-agent
         classic: yes
       when: ansible_distribution == "Ubuntu"
```

Comment:
snap is supported by multiple distros so maybe we should change this to `when: ansible_distribution != "Amazon"` ?

Reply:
I like the idea of that, but in practice I think its safer to stick with a match that we know has snaps. For example, if we want to use the playbook for centos or windows etc